src/components/application-loader/initializers/index.ts

```diff
@@ -21,13 +23,20 @@ export interface InitTask {
   task: Promise<void>
 }
 
-export const createSetUpTaskList = (baseUrl: string): InitTask[] => {
+export const createSetUpTaskList = (frontendAssetsUrl: string, customizeAssetsUrl: string, backendBaseUrl: string): InitTask[] => {
+  setApiUrl({
+    apiUrl: `${ backendBaseUrl }/api/private`
+  })
+
   return [{
     name: 'Load Translations',
-    task: setUpI18n()
+    task: setUpI18n(frontendAssetsUrl)
```

Comment:
It's kind of irritating that you talk here about `frontendAssetsUrl` while inside the i18n-initializer you talk about `frontendBaseUrl`.

Reply:
fixed